infrastructure/gke-billing-export/README.md

```diff
@@ -0,0 +1,124 @@
+# gke-billing-export
+
+Google Kubernetes Engine fine grained billing export. This consists of a single binary that can be run in a Google Cloud project, which with appropriate permissions can scrape the metrics server for many Kubernetes Engine clusters. The metrics are stored in a dedicated BigQuery dataset.
+
+This is not an officially supported Google product.
+
+# Example Output
```

Comment:
This is potentially confusing, because it is showing output from the bq query tool rather than 'billing'.
Can you add a note saying this is data that was inserted by the billing tool?

Reply:
Done.